Mathlib/Data/Finset/Basic.lean

```diff
@@ -3679,7 +3679,8 @@ private lemma pairwiseDisjoint_fibers : Set.PairwiseDisjoint ↑t fun a ↦ s.fi
   fun x' hx y' hy hne ↦ by
     simp_rw [disjoint_left, mem_filter]; rintro i ⟨_, rfl⟩ ⟨_, rfl⟩; exact hne rfl
 
-@[simp] lemma disjiUnion_filter_eq (s : Finset α) (t : Finset β) (f : α → β) :
+-- False positive, it seems
```

Comment:
Can you explain *which* simpNF error this is guarding against?

Reply:
That the statement can't simplify itself.